src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -446,7 +446,9 @@ def launch_fluent(
                     break
                 if start_timeout == 0:
                     LOG.error("The launch process has been timed out.")
```

Comment:
@prmukherj do we need to log the error as well as raise?

Reply:
@seanpearsonuk, the Runtime error msg has been updated with the error log msg and the log has been deleted.